src/Config.js

```diff
@@ -19,13 +19,16 @@ var Config = function (root, configpath) {
     this.initOptions();
     this.initExporters();
     this.initLoaders();
+    this.initRenderers();
     this.initStatics();
     if (!this.configpath) this.ensureDefaultUserConfigPath();
     this.loadUserConfig();
     this.pluginsManager = new PluginsManager(this);  // Do we need back ref?
     this.emit('loaded');
     this.on('server:init', this.attachRoutes.bind(this));
-    this.parsed_opts = {};  // Default. TODO better option management.
+    this.parsed_opts = {
+        renderer: 'carto'
```

Comment:
I'd love to see a more dynamic approach here, but maybe that's not that easy, and given nomnom is not maintained anymore (NodeJS world…) we may be limited. I'll create ticket to remove nomnom to a maintained alternative.